### PR TITLE
Add migration for global settings for old apps

### PR DESCRIPTION
**Pull request by gondar00**
Adds globalsettings block to appDefinition for old apps

### CODE REVIEW

File: server/migrations/1639038616546-UpdateDefinitionsForGlobalSettings.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    const entityManager = queryRunner.manager;
+    const queryBuilder = queryRunner.connection.createQueryBuilder();
+    const appVersionRepository = entityManager.getRepository(AppVersion);
+
+    const appVersions = await appVersionRepository.find();
+
+    for (const version of appVersions) {
+      const definition = version['definition'];
+
+      if (!definition.globalSettings) {
```
Comment (akshaysasidrn) [inline server/migrations/1639038616546-UpdateDefinitionsForGlobalSettings.ts:15]: will throw error when definition is null

File: server/migrations/1639038616546-UpdateDefinitionsForGlobalSettings.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { AppVersion } from '../src/entities/app_version.entity';
+
+export class UpdateDefinitionsForGlobalSettings1639038616546 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    const entityManager = queryRunner.manager;
```
Comment (akshaysasidrn) [inline server/migrations/1639038616546-UpdateDefinitionsForGlobalSettings.ts:6]: can be removed?

**Review by akshaysasidrn — CHANGES_REQUESTED** [review]

File: server/migrations/1639038616546-UpdateDefinitionsForGlobalSettings.ts (hunk 1 above)
Comment (akshaysasidrn) [reply server/migrations/1639038616546-UpdateDefinitionsForGlobalSettings.ts:15]: ignore^

File: server/migrations/1639038616546-UpdateDefinitionsForGlobalSettings.ts (hunk 2 above)
Comment (akshaysasidrn) [reply server/migrations/1639038616546-UpdateDefinitionsForGlobalSettings.ts:6]: ignore ^

File: server/migrations/1639038616546-UpdateDefinitionsForGlobalSettings.ts (hunk 1 above)
Comment (gondar00) [reply server/migrations/1639038616546-UpdateDefinitionsForGlobalSettings.ts]: Added empty check ^^

**Review by akshaysasidrn — APPROVED** [review]

**Pull request merged**